### PR TITLE
[build] Use correct year and copyright owner in license files

### LINKED ISSUE #1389: [license] We should use correct year and name of copyright owner in license file
* 1 We‘ve been not filled the year and copyright owner in license file (https://github.com/ververica/flink-cdc-connectors/blob/master/LICENSE#L189), we should fill the correct year and name of copyright owner 
* 2.Only the project belongs to ASF can use ASF as  the copyright owner, other projects who even use Apache License 2.0 protocol should not use ASF as  the copyright owner, see [1][2]

[1] https://github.com/GoogleCloudPlatform/cloud-spanner-r2dbc/blob/main/cloud-spanner-r2dbc/src/main/java/com/google/cloud/spanner/r2dbc/SpannerType.java#L2
[2] https://github.com/amzn/ion-java/blob/ee2a306d3ecb509973b708afcbf0ddab9dd31887/src/com/amazon/ion/apps/BaseApp.java#L2

### PR DESCRIPTION
**Pull request by leonardBang**
The PR aims to Fix #1389

### CODE REVIEW

File: flink-cdc-e2e-tests/src/test/resources/docker/mysql/my.cnf [hunk 1]
```diff
@@ -1,10 +1,8 @@
...
-# distributed with this work for additional information
-# regarding copyright ownership.  The ASF licenses this file
-# to you under the Apache License, Version 2.0 (the
-# "License"); you may not use this file except in compliance
-# with the License.  You may obtain a copy of the License at
+// Copyright 2022 Ververica
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
```
Comment (Jiabao-Sun) [inline flink-cdc-e2e-tests/src/test/resources/docker/mysql/my.cnf:5]:
```suggestion
# Copyright 2022 Ververica
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
```

File: flink-connector-mongodb-cdc/src/test/resources/ddl/inventory.js [hunk 2]
```diff
@@ -14,7 +12,12 @@
 // under the License.
 
 db.getCollection('products').insertMany([
-{"_id": ObjectId("100000000000000000000101"), "name": "scooter", "description": "Small 2-wheel scooter", "weight": 3.14},
+    {
+        "_id": ObjectId("100000000000000000000101"),
+        "name": "scooter",
+        "description": "Small 2-wheel scooter",
+        "weight": 3.14
+    },
```
Comment (Jiabao-Sun) [inline flink-connector-mongodb-cdc/src/test/resources/ddl/inventory.js:20]: I think we needn't change the format of this line.

**Review by Jiabao-Sun — COMMENTED** [review]
Thanks @leonardBang. I left a few comments.

File: LICENSE [hunk 3]
```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2022 Ververica
```
Comment (wuchong) [inline LICENSE:189]:
```suggestion
   Copyright 2022 Ververica Inc.
```

File: azure-pipelines.yml [hunk 4]
```diff
@@ -1,10 +1,8 @@
-# Licensed to the Apache Software Foundation (ASF) under one or more
-# contributor license agreements.  See the NOTICE file distributed with
-# this work for additional information regarding copyright ownership.
-# The ASF licenses this file to You under the Apache License, Version 2.0
-# (the "License"); you may not use this file except in compliance with
-# the License.  You may obtain a copy of the License at
+# Copyright 2022 Ververica
```
Comment (wuchong) [inline azure-pipelines.yml:1]:
```suggestion
# Copyright 2022 Ververica Inc.
```

**Comment by leonardBang** [comment]
@wuchong @Jiabao-Sun Thanks for the review, I've addressed your comments, please take a look when you have time.

**Review by Jiabao-Sun — APPROVED** [review]

**Pull request merged**